### PR TITLE
report-job-status: add repo to message subject

### PR DESCRIPTION
**Pull request by lastoCHka42**
Снимок экрана 2022-08-05 в 10 41 45

### CODE REVIEW

**Review by NickVolynkin — APPROVED** [review]

File: report-job-status/action.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ runs:
             failedStepsMsg = ""
           }
           
-          let message = `🔴 Job failed:
+          let message = `🔴 Job failed at <a href="${baseUrl}/${{ github.repository }}/">${{ github.repository }}</a>:
```
Comment (NickVolynkin) [inline report-job-status/action.yml:55]: why did you use single braces here?

**Review by ylobankov — CHANGES_REQUESTED** [review]
Please remove the hell in commits.

**Review by ylobankov — COMMENTED** [review]
Please use the 50/72 rule for the commit message.

**Review by ylobankov — APPROVED** [review]
LGTM now, thanks.

**Pull request merged**